pkgs/development/libraries/mesa/default.nix

```diff
@@ -255,14 +264,15 @@ self = stdenv.mkDerivation {
     (lib.mesonEnable "gallium-va" false)
     (lib.mesonEnable "gallium-xa" false)
     (lib.mesonEnable "lmsensors" false)
+  ] ++ lib.optionals enableTeflon [
+    (lib.mesonBool "teflon" true)
```

Comment:
```suggestion
    (lib.mesonBool "teflon" enableTeflon)
```

Reply:
There was something weird with `-Dteflon=false` but I'll check again.

Reply:
Works now, I guess it was unrelated.